cmd/zdb/zdb.c

```diff
@@ -2292,6 +2422,8 @@ dump_label(const char *dev)
 			nvlist_t *vdev_tree = NULL;
 
 			dump_nvlist(config, 4);
+			if (l == 3 && dump_opt['l'] >= 2)
```

Comment:
Not a real problem, but the `l == 3` check which makes sense now will likely cause a tiny conflict down the road.  @richardelling made the excellent suggestion of updating `zdb -l` such that it only dumps one label when they're all the same.